### DISCUSSION TITLE
Option support for proxy download?

### DISCUSSION THREAD

**Discussion by ouyoungs** [Tyrrrz/YoutubeDownloader#564] [Q&A]
Is it possible to add proxy manaly? My network needs a proxy to come out。 looks like this?
![Snipaste_2024-12-15_17-12-17](https://github.com/user-attachments/assets/8f6f381a-0237-4e8b-b378-4f064acad592)

**Comment by Tyrrrz** [comment]
The application should respect the default proxy settings you have set up on your system. Otherwise, you can also configure a proxy for all .NET applications (including YoutubeDownloader) using environment variables: https://learn.microsoft.com/uk-ua/dotnet/api/system.net.http.httpclient.defaultproxy?view=net-8.0#remarks

![image](https://github.com/user-attachments/assets/222e3cd3-bf7f-44bc-b471-d3548d40715c)
